getCounty.py

```diff
@@ -3506,6 +3506,12 @@
             "Sectorul 4",
             "Sectorul 5",
             "Sectorul 6",
+            "Sector 1",
+            "Sector 2",
+            "Sector 3",
+            "Sector 4",
+            "Sector 5",
+            "Sector 6",
```

Comment:
Nu adaugam noi orase deoarece structura asta o folosesc toti scraperisti si vom avea probleme in filtrarea datelor